qemu/tests/cfg/boot_cpu_model.cfg

```diff
@@ -49,9 +49,16 @@
                 - remove_flag:
                     cpu_model_GenuineIntel = "SandyBridge"
                     cpu_model_AuthenticAMD = "Opteron_G3"
-                    cpu_model_flags = ",-sse4a,-xsave,-aes,-sse4.2"
+                    cpu_model_flags = ",-sse4a,-xsave,-aes,-sse4.2,+ht"
+                    cpu_model_flags_AuthenticAMD = ",+cmp_legacy"
                 - cpuid_7_0_ebx_feature:
                     cpu_model_flags = ",+fsgsbase"
                     cpu_model = "SandyBridge"
                 - cpu_host:
                     cpu_model = "host"
+                    cpu_model_flags = 
+                    AuthenticAMD_option_flags = " tsc_deadline_timer"
+                    Opteron_G3_extra_flags = ",-lahf_lm,+3dnow,+pdpe1gb,+3dnowprefetch,+3dnowext"
+                    Opteron_G4_extra_flags = ",-x2apic"
```

Comment:
tsc_deadline_timer is Intel-defined CPU features, not in AMD host and dumpfile, but it is in Guest. So set in AMD_option_flags.